zokrates_stdlib/tests/tests/hashes/utils/256bitsDirectionHelper.zok

```diff
@@ -12,7 +12,7 @@
 // entropy_bits = bitstring.BitArray(bytes=entropy).bin
 // digest_bits = bitstring.BitArray(bytes=digest.compress()).bin
 
-import "hashes/utils/256bitsDirectionHelper.code" as dicrection
+import "hashes/utils/256bitsDirectionHelper" as dicrection
```

Comment:
While I respect some discretion, I think this should be imported and used `as direction`.